### ISSUE TITLE
When is_static=True and no pre-training parameters provided, no error occurs.

### ISSUE THREAD

**Issue by lcy-seso** [PaddlePaddle/Paddle#2069]
Please use this PR https://github.com/PaddlePaddle/models/pull/34/files as an example, in which, some parameters are set `is_static=True` and no pre-training parameters are provided, but it still can run. This is not reasonable and some bugs may happen. 

I suspect whether `is_static=True` is enable, and whether these static parameters are updated during training？

**Comment by lcy-seso** [comment]
@qingqing01 has told me, once a parameter is set `is_static=True`, it will not be updated during the training, but if this is the case, why there casts an error?  how is the static parameter initialized?  What is the training process computing?

**This behavior is dangerous.**

I find the parameter `init_model_path` is invalid and useless for V2 API. Should this be documented?

**Comment by QiJune** [comment]
At first, if the parameter attribute of initialization method is not set, the parameter's value will be set to zero by default.
Second, if is_static=True is set, the gradient buf of the parameter will not be allocated, and the backward of the layer will be skipped.  The parameter value will not be updated.
So, the training process still can run, just some parameter are not updated.

**Comment by lcy-seso** [comment]
But, if a parameter is set to static, it should be explicitly initialized, or cause an error.

**Comment by QiJune** [comment]
Usually, we will set static attribute to some parameter if we want to do some fine-tuning.
You are right, maybe we should  ensure that the static parameter must be explicitly initialized.

**Comment by Canpio** [comment]
Close this issue due to inactivity. please feel free to reopen it if more information is available.

**Comment by lcy-seso** [comment]
This is a bug that should be fixed, so I reopen it.
